torch/csrc/jit/codegen/cuda/transform_replay.cpp

```diff
@@ -645,9 +639,15 @@ bool TransformPropagator::replayPasC(
   if (producer_tv == starting_tv) {
     return false;
   }
```

Comment:
Sorry, these changes snuck in but are important. Before I wasn't tracking how far something was replayed in transform propagator, and this would cause issues when something wasn't fully replayed. The logic would think tensors were fully replayed even if they weren't and therefor later propagation paths wouldn't proceed, and certain tensors would just never be transformed correctly.